vertical-pod-autoscaler/pkg/recommender/input/cluster_feeder.go

```diff
@@ -321,6 +327,8 @@ func (feeder *clusterStateFeeder) LoadVPAs() {
 		selector, conditions := feeder.getSelector(vpaCRD)
 		glog.Infof("Using selector %s for VPA %s/%s", selector.String(), vpaCRD.Namespace, vpaCRD.Name)
 
+		conditions = append(conditions, feeder.validateTargetRef(vpaCRD)...)
```

Comment:
As discussed offline, this will set the condition but will not clear the recommendation.

Reply:
Done.